handler/src/test/java/io/netty/handler/ipfilter/IpSubnetFilterTest.java

```diff
@@ -23,60 +23,68 @@
 import io.netty.channel.embedded.EmbeddedChannel;
 import io.netty.util.internal.SocketUtils;
 import org.junit.Assert;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
```

Comment:
@skyguard1 please move the static imports in a separate "block"

Reply:
I've done it